src/AbstractEvent.php

```diff
@@ -19,7 +19,7 @@
     protected $emitter;
 
     /**
-     * {@inheritdoc}
+     * @inheritdoc
```

Comment:
I'm just confused as to why the php documentor lead would tell you something that's conflicting with the spec coming from php documentor?